.github/workflows/contributors-record.yml

```
@@ -0,0 +1,32 @@
+# Give your test a name!
+name: ContributorsRecord
+
+# Decide when to run the tests
+#
+# This configuration sets the test to run on pushes to master
+# and on pull requests that are opened to master
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+# Set up the Continuous Integration job
+jobs:
+  ContributorsRecord:
+    # Run on the latest Ubuntu distribution
+    runs-on: ubuntu-latest
+    # This section collects together the steps involved in running the test
+    steps:
+    # Checkout the repository. Relies on another GH-Action.
+    - uses: actions/checkout@v2
+    # Run the script to check file sizes
+    - name: Create latest contributors record
+      run: |
+        ./tests/contributors.sh
+    - uses: EndBug/add-and-commit@v5
+      with:
+        add: ./book/website/afterword/contributors-record.md
+        message: 'Add contributors-record.md to Afterword'
```

```suggestion
    - uses: EndBug/add-and-commit@v5
      if: github.event.pull_request
      with:
        add: ./book/website/afterword/contributors-record.md
        message: 'Add contributors-record.md to Afterword'
        author_name: CI User
        author_email: ci-user@github.local
        branch: github.head_ref  # This can only be used when triggered by a PR
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Just adding some extra fields that will make sure this commit happens properly and doesn't cause git errors.

---

Added it locally.